compose/config/config.py

```diff
@@ -133,6 +144,24 @@ def find(base_dir, filenames):
         [ConfigFile.from_filename(f) for f in filenames])
 
 
+def get_config_version(config_details):
+    def get_version(config):
+        validate_top_level_object(config)
+        return config.config.get('version')
```

Comment:
Could we set a default version of `1` here, so that the rest of the code dones't need to know about the "no version is version 1" assumption?  We can keep that logic in this one place. 

I guess that will require a `git mv` of the fields schema file.